x-pack/test/cases_api_integration/security_and_spaces/tests/trial/configure/get_connectors.ts

```diff
@@ -184,5 +185,19 @@ export default ({ getService }: FtrProviderContext): void => {
         },
       ]);
     });
+
+    it('should return unauthorized error when API tag not present', async () => {
```

Comment:
nit: `should get 401 when trying to get the case connectors with no connectors access`